src/components.jsx/ArticleCard.jsx

```diff
@@ -34,10 +34,16 @@ const ArticleCard = ({ article }) => {
       </CardBody>
       <Divider />
       <CardFooter className="article-footer">
-          <Button variant="solid" colorScheme="teal">
-            Read More
-          </Button>
-          <Text color='teal'>{articleDate}</Text>
+        <Button
+          variant="solid"
+          colorScheme="teal"
+          onClick={() => {
+            window.location.replace(`/articles/${article.article_id}`);
+          }}
+        >
+          Read More
+        </Button>

```

Comment:
This really should be an anchor tag (react router Link tag) as you are navigating to a new page. A button is designed to interact with the page so semantically doesn't make sense here.